sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -251,6 +251,9 @@ def get_queue_sender(self, queue_name, **kwargs):
 
         :param str queue_name: The path of specific Service Bus Queue the client connects to.
         :rtype: ~azure.servicebus.ServiceBusSender
+        :keyword str client_identifier: A string-based identifier to uniquely identify the sender instance.
```

Comment:
I thought keyword parameters need to be documented before the `rtype` - but maybe that's not actually necessary if the docs are rendering correctly?